### PR TITLE
completed the missing functions

### LINKED ISSUE #2: Complete the functions left undone & fill up missing return values

### PR DESCRIPTION
**Pull request by abhishek-mishra17**
Closes #2

### CODE REVIEW

File: air_sim_func.c [hunk 1]
```diff
@@ -2,7 +2,14 @@
 #define ARRIVE 0
 #define DEPART 1
 //fill up
+#include <conio.h>
+#include <stdlib.h>
+#include <ctype.h>
+#include <math.h>
+#include <time.h>
+#include <limits.h>
```
Comment (chinmayishetty359) [inline air_sim_func.c]: Not required as we added that in air_sim.h

File: air_sim.h [hunk 2]
```diff
@@ -9,7 +9,7 @@
 #define MAX 15
 #define ARRIVE 0
 #define DEPART 1
-#define RAND_MAX 
+#define RAND_MAX  1
```
Comment (chinmayishetty359) [inline air_sim.h]: Not required

Comment (abhishek-mishra17) [reply air_sim.h]: Ah sorry actually it was showing an error in my editor , thats why I had added it.

Comment (abhishek-mishra17) [reply air_sim.h]: I'll change it now

**Comment by abhishek-mishra17** [comment]
@chinmayishetty359 I have made the changes

File: air_sim.h (hunk 2 above)
Comment (chinmayishetty359) [reply air_sim.h]: Yeah, pls complete initairport func also so u can get more points!!!

File: air_sim_main.c [hunk 3]
```diff
@@ -7,9 +7,10 @@
 void main()
 {
     srand(time(NULL));
+    struct airport a;
```
Comment (chinmayishetty359) [inline air_sim_main.c:10]: Should be a pointer as in air_sim_func

File: air_sim_main.c [hunk 4]
```diff
@@ -7,9 +7,10 @@
 void main()
 {
     srand(time(NULL));
+    struct airport a;
     int i, pri, curtime, endtime;
     double expectarrive, expectdepart;
-    struct plane *temp;
+    struct plane temp;
```
Comment (chinmayishetty359) [inline air_sim_main.c]: Even this should be a pointer

**Comment by chinmayishetty359** [comment]
Sorry i didn't observe this before..pls make th eabove changes.

**Review by chinmayishetty359 — COMMENTED** [review]
complete the initairport func too!!

**Comment by abhishek-mishra17** [comment]
@chinmayishetty359 made the required changes

**Comment by chinmayishetty359** [comment]
!bounty 45

**Comment by chinmayishetty359** [comment]
Brownie points for the pointer changes in all the functions

**Pull request merged**